students/JeroenSweerts/Lesson4/kata.py

```diff
@@ -0,0 +1,40 @@
+import os
+import random
+#get path of current work directory
+os.chdir(os.path.dirname(__file__))
+cwd = os.getcwd()
+
+#open the text file and read it into a list of words
+f = open(cwd + '\sherlock.txt','rb')
+text = f.read()
+words = text.split()
+words = [str(i) for i in words]
+f.close()
```

Comment:
it is preferred to use `with` block when working with files
http://www.pythonforbeginners.com/files/with-statement-in-python

so this would look like:
```
with open(os.path.join(cwd, filename), 'r'):
    text = f.read()
words = text.split()

```
notice in above only read() is under `with` block and the use of `os.path.join` you want to avoid manually creating filepaths.